serverlessworkflow/modules/ROOT/assets/images/job-services/Job-Service-Generic-Diagram.drawio

```diff
@@ -0,0 +1,105 @@
+<mxfile host="app.diagrams.net" modified="2023-07-07T11:15:47.365Z" agent="Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36" etag="qxEMbWAcp6-76LZQZ2AI" version="21.6.1" type="google">
```

Comment:
Is this file included in the doc?

Reply:
yep, not "ideal", but otherwise it's very difficult to do changes in the diagrams in the future. So I started pushing them too.